scripts/extract_js_texts.js

```diff
@@ -34,7 +33,10 @@ const source_strings = {};
 const ignored_list   = {};
 const invalid_list   = {};
 
-let this_app_name;
+let this_app_name, emphasize;
+import('emphasize').then(module => {
```

Comment:
why do we need to import it like this?

Reply:
explained here: https://github.com/deriv-com/smarttrader/pull/617#discussion_r1533479610